python/sidecar/typing.py

```diff
@@ -198,7 +198,11 @@ class S2SearchResult(RefStudioModel):
 class SearchResponse(RefStudioModel):
     status: ResponseStatus
     message: str
-    results: list[S2SearchResult] 
+    results: list[S2SearchResult]
+
+
+class ServeResponse(RefStudioModel):
+    pass
```

Comment:
What's this @danvk? I don't see this `ServeResponse` type being used in the API.